src/utils/findpackage.rs

```diff
@@ -50,6 +50,33 @@ fn handle_config_package_tst() {
 static SPECIAL_PACKAGE_PATTERN: LazyLock<regex::Regex> =
     LazyLock::new(|| regex::Regex::new(r"([a-zA-Z_\d\-]+)-(\d+(\.\d+)*)").unwrap());
 
+static CMAKE_PREFIXES: LazyLock<Vec<String>> = LazyLock::new(|| {
+    let Ok(command) = std::process::Command::new("cmake")
+        .arg("--system-information")
+        .output()
+    else {
+        return vec![];
+    };
+    let output = String::from_utf8(command.stdout).unwrap();
```

Comment:
there is still some problems... on windows platform.. will this be ok?

Reply:
I'm not sure, I didn't think about windows when creating this